test/Macros/print_clang_expand_on_imported.swift

```diff
@@ -7,8 +7,10 @@
 // Build the macro library to give us access to AddAsync.
 // RUN: %target-swift-frontend -swift-version 5 -emit-module -o %t/macro_library.swiftmodule %S/Inputs/macro_library.swift -module-name macro_library -load-plugin-library %t/%target-library-name(MacroDefinition)
 
-// RUN: %target-swift-ide-test(mock-sdk: %clang-importer-sdk) -print-module -print-implicit-attrs -source-filename %s -module-to-print=CompletionHandlerGlobals -I %t -function-definitions=false -load-plugin-library %t/%target-library-name(MacroDefinition) -import-module macro_library -enable-experimental-feature MacrosOnImports > %t/imported.printed.txt
+// RUN: %target-swift-ide-test(mock-sdk: %clang-importer-sdk) -print-module -print-implicit-attrs -source-filename %s -module-to-print=CompletionHandlerGlobals -I %t -function-definitions=false -load-plugin-library %t/%target-library-name(MacroDefinition) -import-module macro_library -enable-experimental-feature MacrosOnImports > %t/imported.printed.txt 2> %t/stderr.log
 // RUN: %FileCheck -input-file %t/imported.printed.txt %s
+// RUN: echo "" >> %t/stderr.log
+// RUN: %FileCheck -check-prefix DIAGS -input-file %t/stderr.log %s
```

Comment:
Is it not possible to instead have `swift-ide-test` return an error code if the input has errors? That seems like a simpler solution. Not sure what effect they have, but it sounds like `--test-input-complete` or ` --typecheck` could be useful.